internal/bminventory/inventory_test.go

```diff
@@ -2305,6 +2327,108 @@ var _ = Describe("cluster", func() {
 					})
 					Expect(reply).Should(BeAssignableToTypeOf(common.NewApiError(http.StatusBadRequest, errors.Errorf("error"))))
 				})
+
+				It("should return error when both cnv and lvm operator enabled", func() {
+					mockVersions.EXPECT().GetReleaseImage(gomock.Any(), gomock.Any()).Return(common.TestDefaultConfig.ReleaseImage, nil).Times(1)
+					mockOperatorManager.EXPECT().GetSupportedOperatorsByType(models.OperatorTypeBuiltin).Return([]*models.MonitoredOperator{&common.TestDefaultConfig.MonitoredOperator}).Times(1)
+					mockOperatorManager.EXPECT().ResolveDependencies(gomock.Any()).
+						DoAndReturn(func(operators []*models.MonitoredOperator) ([]*models.MonitoredOperator, error) {
+							return operators, nil
+						}).Times(1)
+					mockEvents.EXPECT().SendClusterEvent(gomock.Any(), eventstest.NewEventMatcher(
+						eventstest.WithNameMatcher(eventgen.ClusterRegistrationFailedEventName),
+						eventstest.WithSeverityMatcher(models.EventSeverityError))).Times(1)
+					mockLVMGetOperatorByName("lvm")
+					mockCNVGetOperatorByName("cnv")
+
+					clusterParams := getDefaultClusterCreateParams()
```

Comment:
It will be better to add clusterParams.OpenshiftVersion = swag.String("4.11.0") and not count that the default version will be < 4.12